pkg/metrics/metrics.go

```diff
@@ -31,10 +31,17 @@ const (
 	backupAttemptCount          = "backup_attempt_total"
 	backupSuccessCount          = "backup_success_total"
 	backupFailureCount          = "backup_failure_total"
+	backupSecondsCount          = "backup_seconds_total"
```

Comment:
Suggest re-naming to this `backupDurationSeconds = "backup_duration_seconds"`.
The convention that I've seen is I count 'events' and then report them as the total number of events, seen in the look back time window.
In this case, what we are capturing is how long it took to run the backup which is not an event. It is kinda similar to the backup size.

Reply:
+1